krb525_convert.c

```diff
@@ -291,7 +291,7 @@ get_krb525_endpoints(krb5_context context, short port, char *realm, krb525_endpo
 	}
 #endif
 
-	if (!krb525_hosts || !krb525_hosts[0]) {
+	if (!krb525_hosts || !krb525_hosts[0] || strlen(krb525_hosts[0]) == 0) {
```

Comment:
if (!krb525_hosts || !krb525_hosts[0] || !*krb525_hosts[0])

sounds more readable to me

Reply:
Agree. Amended.